index.html

```diff
@@ -0,0 +1,316 @@
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, 
+                    don't hestiate to contact me.</p>
+
+                <div id="headline-social-media">
+                    <ul class="social-media">
+                        <li class="social-media-icon"><a href="http//:github.com/francis450"><img src="img/social-media-icons/github.svg"></img></a></li>
+                        <li class="social-media-icon"><a href="http//:linkedin/in/njoroge-francis"><img src="img/social-media-icons/linkedin.svg"></img></a></li>
+                        <li class="social-media-icon"><a href="http//:evangelist"><img src="img/social-media-icons/angelist.svg"></img></a></li>
+                        <li class="social-media-icon"><a href="http//:twitter.com/nkamandefrancis"><img src="img/social-media-icons/twitter.svg"></img></a></li>
+                        <li class="social-media-icon"><a href="http//:medium.com"><img src="img/social-media-icons/medium.svg"></img></a></li>
+                    </ul>
+                </div>
+                
+                <div class="headline-to-bottom-icon">
+                    <a class="bottom-icon-container" href="#works"><img src="img/down.svg"></img></a>
+                </div>
+            </section>
+        </div>
+
+        <!-- Recent Works Section -->
+        <section id="works">
+            <div id="works-title">
+                <h2 class="works-title-text">My Recent Work</h2>
+            </div>
+
+            <div id="work-cards">
+                <section class="work-cards-row">
+                    <!-- card1 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+
+                    <!-- card2 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
```

Comment:
Kindly use an h3 here as it is directly under an h2. it obeys the hx or header hierarchy rule and it is semantic hence will aid user accessibility for screen readers😉

Reply:
Fixed.